core/amber/src/main/scala/edu/uci/ics/texera/web/model/jooq/generated/Indexes.java

```diff
@@ -36,6 +37,8 @@ public class Indexes {
     public static final Index FILE_UID = Indexes0.FILE_UID;
     public static final Index FILE_OF_PROJECT_PID = Indexes0.FILE_OF_PROJECT_PID;
     public static final Index FILE_OF_PROJECT_PRIMARY = Indexes0.FILE_OF_PROJECT_PRIMARY;
+    public static final Index KEYWORD_DICTIONARY_PRIMARY = Indexes0.KEYWORD_DICTIONARY_PRIMARY;
+    public static final Index KEYWORD_DICTIONARY_UID = Indexes0.KEYWORD_DICTIONARY_UID;
```

Comment:
please check the generated files again. I think the keyword dictionary related code should not be added here.

Reply:
keyword_dictionary is on master again...